wisp-lease-testing/src/main/kotlin/wisp/lease/FakeLeaseManager.kt

```diff
@@ -7,25 +7,30 @@ import java.util.concurrent.ConcurrentHashMap
  * default, a fake lease is considered held, but it can be explicitly marked as not held if desired
  */
 open class FakeLeaseManager : LeaseManager {
-    private val leasesHeldElsewhere = ConcurrentHashMap<String, Int>()
-    private val leases = ConcurrentHashMap<String, FakeLease>()
+  private val leasesHeldElsewhere = ConcurrentHashMap<String, Int>()
+  private val leases = ConcurrentHashMap<String, FakeLease>()
 
-    override fun requestLease(name: String): Lease {
-        return leases.computeIfAbsent(name) {
-            FakeLease(name, this)
-        }
+  override fun requestLease(name: String): Lease {
+    return leases.computeIfAbsent(name) {
+      FakeLease(name, this)
     }
+  }
 
-    fun isLeaseHeld(name: String) = !leasesHeldElsewhere.containsKey(name)
-
-    fun markLeaseHeld(name: String) {
-        leasesHeldElsewhere.remove(name)
-        (requestLease(name) as FakeLease).acquire()
+  override fun releaseAll() {
+    leases.forEachValue(1) {
```

Comment:
what does `forEachValue(1)` mean?

Reply:
requires a `parallelism` -- in this case it's just a standard `forEach`